### ISSUE TITLE
"Required" red text color contrast does not pass WCAG 2.2 AAA

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#95526]
## Issue Description
This issue was identified in the [accessibility audit of the review & confirmation pages ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94614)of the 526 form.

The color contrast against the grey background is 6.12:1.
The color contrast against the white background is 6.98:1. It should be 7:1 for WCAG 2.2 level AAA. 

![Image](https://github.com/user-attachments/assets/76f8091a-5190-4aff-b1c7-8d8ca318c451)
![Image](https://github.com/user-attachments/assets/eafe37b8-3cda-48be-8ba8-4357acb78ece)

To fix: Increase the font size to 18, bold the text, or darken the red to get a 7:1 contrast. This is a AAA requirement, so it would just be a minor improvement. This contrast currently passes AA.

[**Expected severity**](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric): a11y-defect-4
**Page**: review-and-submit
**Environment**: any screenreader/browser combo
**Estimated Level of Effort to fix**: low
**Design system component involved**: Yes
**Relevant WCAG success criteria**: [1.4.6 Contrast (Enhanced) Level AAA](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_overview#contrast-enhanced)

## Tasks
- [ ] Increase the font size to 18, bold the text, or darken the red. 
- [ ] Please update the "Fix Status" column in the [526 Accessibility Audit issues tracking doc](https://docs.google.com/spreadsheets/d/1paNxD798NSMrjZeUe9WwVQ4xRdl439ypKyPDPW3CXIo/edit?gid=2034611899#gid=2034611899) when ticket is started and completed.

## Acceptance Criteria
- [ ] Screenreader users hear that their change was successful.
- [ ] Accessibility Audit issue tracking doc is updated with ticket status
